images/macos/toolsets/toolset-10.15.json

```diff
@@ -21,6 +21,13 @@
         ],
         "bundle-default": "latest",
```

Comment:
probably we should hardcode 6_12_1 as a latest bundle because [Xamarin.iOS 14.2 requires Xcode 12.1](https://docs.microsoft.com/en-ie/xamarin/ios/release-notes/14/14.2) or higher but we have Xcode 12.0.1 as a default one, and I am afraid we can break default Xamarin.iOS/default Xcode pair again. 
@maxim-lobanov what do you think?

Reply:
Done.